prov/efa/src/rxr/rxr_domain.c

```diff
@@ -190,10 +190,9 @@ int rxr_domain_open(struct fid_fabric *fabric, struct fi_info *info,
 				  	util_domain.domain_fid);
 
 	/* Open shm provider's access domain */
-	if (rxr_env.enable_shm_transfer) {
-		efa_fabric = container_of(fabric, struct efa_fabric,
-					util_fabric.fabric_fid);
-
+	efa_fabric = container_of(fabric, struct efa_fabric,
+				util_fabric.fabric_fid);
```

Comment:
indentation off here. Also because the line shifted to the left, so there is no need to be two line.

Reply:
Fixed! Thanks.